packages/chargebee/src/action/index.stories.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import Action from './index';
+
+export default { title: 'Action', component: Action };
+
+export const Default = () => (
+  <Action
+    site="site-name"
+    onCall={action('Call')}
+  >
+    Click me!
```

Comment:
make it a proper button with ui-button?

Reply:
Why? Will need to write styles for that

Reply:
No need for any custom styles. nebenan-ui-kit is imported in all storybook stories